source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,30 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-07-14
+
+### Highlights
+
+#### Granular Data Retention Policies
+ - A ``data_retention`` type job can now be run even if the global policy is disabled. The granular (i.e. team and channel-specific) policies will be executed when the data retention job is run. Please note there is a known issue where deleted posts get displayed in channels without new activity after the retention job is run.  This issue is tracked with [this ticket](https://mattermost.atlassian.net/browse/MM-36574).
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a **Do Not Disturb** availability submenu with the options to disable notifications for 30min, 1h, 2h, Tomorrow, or Custom.
+ - Markdown formatting is now stripped from push notifications.
+ - Improved performance of components that show reactions on posts.
+ - Improved performance of certain components when viewing non-Direct Message channels.
+ - Added minor improvements to performance of messages posted in the right-hand side.
+ - Added icons to apps in the Marketplace.
+ - Apps can now add arbitrary markdown in between fields on forms.
+ - Added support for markdown in apps forms, interactive dialogs field descriptions, errors, and slash commands.
+
+### Known Issues
+ - Sometimes an "Unable to get role" error appears when changing a channel member role in **System Console > User Management > Channels**.
```

Comment:
Propose adding a link to CRT docs known issues section in every cloud and on-prem changelog until we release the feature in GA

Reply:
@amyblais ^